aziot/src/init.rs

```diff
@@ -1065,6 +1065,7 @@ fn write_file(
     Ok(())
 }
 
+#[cfg(debug_assertions)]
```

Comment:
Is there any reason why the tests wouldn't work in release mode?

Reply:
https://github.com/Azure/iot-identity-service/pull/57/commits/e36e1334cd8c4a37f4506e5dcb3c1845e3c3dd24

>restrict argv[1] fallback to debug builds.

Reply:
sorry, I don't follow :grimacing: 

What does this test block have to do with `argv[1]`?

Reply:
Oh my bad, you're asking about the init test.

The answer to that is https://github.com/Azure/iot-identity-service/blob/becac6833628aece78cddffe9052f5c81d624d92/aziot/src/init.rs#L170-L172

Reply:
Ahhh, I see.

That's somewhat non-obvious though, maybe add a comment on the test module explaining why it's disabled in release mode?

Reply:
I'll just move the check to the `run()` function.